contrib/dex/README.md

```diff
@@ -0,0 +1,65 @@
+## Exposing DEX service
+After the cluster is up, you have to manually expose dex service using a ELB or Ingress.
```

Comment:
@c-knowles Yes, I think it makes more sense to put it under `addons`. I'll make the changes.
@mumoshu I prefer doing this part in a separate PR. Right now the TLS termination is handled by ELB or Ingress and we need to configure the certificates for the ELB. Kubernetes + Amazon Certificate Manager makes this easy. Of course I can enable `https` on `dex`but then lose the possibility to use the certificates provided by Amazon. Have to think more on this part.